rust/rbac-registration/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "rbac-registration"
 description = "Role Based Access Control Registration"
 keywords = ["cardano", "catalyst", "rbac registration"]
-version = "0.0.7"
+version = "0.1.7"
```

```suggestion
version = "0.0.8"
```
Same reason (permanently unstable for now)